jolly-jellyfish/src/page_maker/templates/page_maker/base.html

```diff
@@ -0,0 +1,41 @@
+{% load static %}
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    {% if webpage %}
+      <title>{{ webpage.user_title }}</title>
+    {% else %}
+      <title>Early Internet Page Maker</title>
+    {% endif %}
+    {% block header %}
+    {% endblock %}
+  </head>
+  <body>
+    <header>
+      Hello!
+      {% if user.is_authenticated %}
+        <a href="{% url 'user-detail' pk=user.id %}"> {{ user.username }}</a>,
+        <a href="{% url 'logout' %}">logout</a>
+      {% else %}
+        <a href="{% url 'login' %}">Login</a>
+        <a href="{% url 'register' %}">Register</a>
+      {% endif %}
+      <a href="{% url 'home' %}">home</a>
+    </header>
+
+    <main>
+      {% if error %}
+      <div>
+        <p>{{ error.text }}</p>
+      </div>
+      {% endif %}
```

Comment:
What sort of errors would these be? Could they be moved to after the content? Styling concerns...

Reply:
I just dropped it in before creating views, but I will remove it since it's not used.